### PR TITLE
[237 ]Fix toast message/urls

### LINKED ISSUE #237: Token Approval Notification link shows Etherscan but links to GP Explorer

### PR DESCRIPTION
**Pull request by W3stside**
Closes #237 

There was a fix a while back to fix Explorer vs Etherscan links/messages in toast but it wasn't considering `approve` transactions which were pointing at GP Explorer

Also fixes `mock/orders` to allow testing propery. Can be reenabled to see the correct links. To test the `approve` toast you have to make an approve tx

I'd like this to be in #247 @alfetopito @anxolin

### CODE REVIEW

File: src/custom/components/Popups/TransactionPopupMod.tsx [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 import styled, { ThemeContext } from 'styled-components'
 import { useActiveWeb3React } from 'hooks'
-import { TYPE } from 'theme'
-// import { ExternalLink } from 'theme'
-// import { getEtherscanLink } from 'utils'
+import { TYPE, ExternalLink } from 'theme'
 import { AutoColumn } from 'components/Column'
 import { AutoRow } from 'components/Row'
-import { ExplorerLink } from '../ExplorerLink'
+import { getEtherscanLink, getExplorerLabel } from 'utils'
```
Comment (anxolin) [inline src/custom/components/Popups/TransactionPopupMod.tsx:8]: I think the solution should be to make `ExplorerLink` to be agnostic of the underlying explorer we use. 

That was certainly the intention. I'll try to make an alternative proposal

**Review by anxolin — COMMENTED** [review]
If this is a bug, maybe we should fix it the release branch instead.

File: src/custom/components/Popups/TransactionPopupMod.tsx (hunk 1 above)
Comment (W3stside) [reply src/custom/components/Popups/TransactionPopupMod.tsx:8]: it is pointing at #247  now - that was my intention (hence the description)

**Comment by anxolin** [comment]
@W3stside pls consider my PR, I try to supersede this one https://github.com/gnosis/gp-swap-ui/pull/254

The idea is to use the Explorer component, no matter which explorer is underneath. This is abstracted away. We don't need ot add the logic of deciding where to point in several places

**Pull request closed without merging**